snakemake/cli.py

```diff
@@ -1482,12 +1463,8 @@ def get_argument_parser(profiles=None):
         "allowing to e.g. send notifications in the form of e.g. slack messages or emails.",
     )
     group_behavior.add_argument(
-        "--log-service",
-        default=None,
-        choices=["none", "slack", "wms"],
-        help="Set a specific messaging service for logging output."
-        "Snakemake will notify the service on errors and completed execution."
-        "Currently slack and workflow management system (wms) are supported.",
+        "--monitoring-provider",
+        help="Specify a custom monitoring provider, available via an monitoring plugin: snakemake_monitoring_<name>",
```

Comment:
```suggestion
        help="Specify a custom monitoring provider, available via an monitoring plugin: snakemake-monitoring-plugin-<name>",
```

Reply:
Thinking about it, why not just `snakemake-monitor-plugin-...` (without the ing)?

Reply:
I'm not sure tbh - I think b/c I had "monitoring-provider" stuck in my head. I like `snakemake-monitor-plugin` better, I'll make that change.